src/Pester.Runtime.ps1

```diff
@@ -59,6 +59,20 @@ $script:SessionStateInternalProperty = [System.Management.Automation.SessionStat
 $script:ScriptBlockSessionStateInternalProperty = [System.Management.Automation.ScriptBlock].GetProperty('SessionStateInternal', $flags)
 $script:ScriptBlockSessionStateProperty = [System.Management.Automation.ScriptBlock].GetProperty("SessionState", $flags)
 
+function Get-StackTraceLanguage {
+    $err = try { throw "err" }  catch { $_ }
+    $firstFrame = ($err.ScriptStackTrace -split [System.Environment]::NewLine, 2 )[0]
+    if ($firstFrame -match "(?<at>.[^G]?)\s+Get-StackTraceLanguage(?<separator>.).+:\s+(?<line>.\S+)\s\d+") {
+        @{
+            At        = $Matches["at"]
+            Separator = $Matches["separator"]
+            Line      = $Matches["line"]
+        }
+    }
+}
```

Comment:
Changed your code to parse it from exception that we throw, as originally suggested. This should be cheaper and easier to replicate than relying on internal resource names because we rely on public api.